rest/testdata/generic/openapi.yml

```diff
@@ -49,11 +49,11 @@ paths:
               schema:
                 $ref: "#/components/schemas/response_object_type"
 
-  /v2/fakeresource/{id}:
+  /v2/fakeresource/{resourceId}:
```

Comment:
I've updated the testdata to reflect cases where endpoint paths may sometimes have a path param that is not called "id" which will test the code path for mapping "id-like" path params to the resource's id.